src/wrapper/strings/java_str.rs

```diff
@@ -79,36 +100,35 @@ impl<'local, 'other_local: 'obj_ref, 'obj_ref> JavaStr<'local, 'other_local, 'ob
         Ok(())
     }
 
-    /// Get a [JavaStr] from a [JNIEnv] and a [JString].
-    /// You probably want [JNIEnv::get_string] instead of this method.
-    pub fn from_env(env: &JNIEnv<'local>, obj: &'obj_ref JString<'other_local>) -> Result<Self> {
-        Ok(unsafe {
+    pub(crate) unsafe fn from_env_totally_unchecked(
```

Comment:
it's internal api, so it's ok either way, but maybe don't need to say that it's 'totally' unchecked, if there's no naming ambiguity or another api that is also unsafe but with more checks.

Reply:
Well, it's called from `JNIEnv::get_string_unchecked`, which is actually not entirely unchecked ([it does a null check](https://github.com/jni-rs/jni-rs/blob/dccd7f94bfff2d804cd23064276fe6ad0e848841/src/wrapper/jnienv.rs#L1877)).

But `JavaStr::get_env_totally_unchecked` *also* isn't entirely unchecked ([it calls](https://github.com/jni-rs/jni-rs/blob/dccd7f94bfff2d804cd23064276fe6ad0e848841/src/wrapper/strings/java_str.rs#L69) `GetStringUTFChars`, which [does unspecified checks of its own](https://docs.oracle.com/en/java/javase/11/docs/specs/jni/functions.html#getstringutfchars)), so…yeah, this is a misnomer too. 🤷‍♂️

I didn't want to leave the name as `from_env` because it would give people confusing compiler errors (“method `from_env` is private”, not “no method named `from_env` found”) when they migrate from an old version of this library. For the same reason, I didn't want to rename it to `new`. I guess I could have named it `from_env_private`?